examples/workload-multiple-secrets/k8s/Identity.yaml

```diff
@@ -22,5 +22,5 @@ spec:
     matchLabels:
       app.kubernetes.io/name: aegis-workload-demo
   workloadSelectorTemplates:
-  - "k8s:ns:{{ .PodMeta.Namespace }}"
```

Comment:
interpolating namespace and service account like this is meaningless; if we are going to enforce it, they need to be hard-coded instead.